.github/workflows/Ci-raiwigets-python-typescript.yml

```diff
@@ -77,5 +77,27 @@ jobs:
 
       - name: Run tests
         run: |
-          pytest --durations=10
+          pytest --durations=10 --doctest-modules --junitxml=junit/test-results.xml --cov=${{ matrix.packageDirectory }} --cov-report=xml --cov-report=html
```

Comment:
I think you can add examples back if you remove the "--doctest-modules" parameter here